modules/ROOT/pages/mmp-concept.adoc

```diff
@@ -141,6 +141,63 @@ include::example$mmp-concept-config.xml[]
 If the `skipDeploymentVerification` parameter is not present, the default value is false. +
 This feature is available in plugin version 3.2.5 and later.
 
+[[configure-shared-libraries]]
+== Configure Shared Libraries
+
+Configure shared libraries when you need to make a dependency visible to your application's connectors. To configure a dependency as a shared library, add the following elements in your Mule Maven plugin configuration inside your project's `pom.xml` file:
+
+[source,xml,linenums]
+----
+<sharedLibraries>
+  <sharedLibrary>
+    <groupId>${dependencyGroupID}</groupId>
+    <artifactId>${dependencyArtifactID}</artifactId>
+  </sharedLibrary>
+</sharedLibraries>
+----
+
+The following example configures the Database connector in a project, and also configures the MySQL driver as a shared library to be used by the connector:
+
+[source,xml,linenums]
+----
+<build>
+  <plugins>
+    <!-- Mule Maven Plugin configuration -->
+    <plugin>
+    	<groupId>org.mule.tools.maven</groupId>
+    	<artifactId>mule-maven-plugin</artifactId>
+    	<version>${mule.maven.plugin.version}</version>
+    	<extensions>true</extensions>
+    	<configuration>
+    	<sharedLibraries>
+        <!-- MySQL driver dependency configured as a shared library -->
+        <sharedLibrary>
+          <groupId>mysql</groupId>
+          <artifactId>mysql-connector-java</artifactId>
+        </sharedLibrary>
+      </sharedLibraries>
+      </configuration>
+    </plugin>
+  </plugins>
+</build>
+
+</dependencies>
+  <!--Databse Connector dependency -->
+  <dependency>
+    <groupId>org.mule.connectors</groupId>
+    <artifactId>mule-db-connector</artifactId>
+    <version>1.5.0</version>
+    <classifier>mule-plugin</classifier>
+  </dependency>
+  <!-- MySQL driver dependency -->
+  <dependency>
+    <groupId>mysql</groupId>
+    <artifactId>mysql-connector-java</artifactId>
+    <version>5.1.33</version>
+  </dependency>
+</dependencies>
+----
+
```

Comment:
we should add a warning here and tell readers that the sharedLibraries will be shared to ALL connectors. That can cause issues if the dependency they are sharing has different versions of the same classes from other connectors (or their dependencies).

With this in mind, we should mention a different way of adding external libraries to a connector from an application that works a little differently than this: <additionalPluginDependencies>. I will explain it over Slack